### ISSUE TITLE
Proposer de supprimer les proches dans la foulée de la suppression d'un usager

### ISSUE THREAD

**Issue by yaf** [betagouv/rdv-solidarites.fr#2289]
## Actuellement

Lorsque l'on supprime un usager, ces proches restent.

## Objectif

Proposer de supprimer les proches dans la foulée (ou le faire automatiquement ?)

**Comment by n-b** [comment]
Ça me parait pas complètement `.S`, ça tire toute la pelote de #1670. Ce qu’on peut faire _rapidement_, c’est poser la question au moment de supprimer un usager, ou plutôt: dans la fiche usager, si celui-ci est responsable de proches, remplacer le bouton “Supprimer” par un menu avec deux options: 

```
Supprimer…
| - Supprimer le compte de Patricia Duroy et de ses proches (Joséphine)
| - Supprimer uniquement le compte de Patricia Duroy

 ```

**Comment by yaf** [comment]
Oui, c'est à ça que je pensais... Merci ! C'est plus étoffé que ce que j'ai noté.
